sections/featured-product.liquid

```diff
@@ -32,7 +32,7 @@
 
 <section class="color-{{ section.settings.color_scheme }} {% if section.settings.secondary_background %}background-secondary{% else %}gradient{% endif %}">
   <div class="page-width section-{{ section.id }}-padding">
-    <div class="featured-product product grid grid--1-col gradient color-{{ section.settings.color_scheme }} {% if product.media.size > 0 %}grid--2-col-tablet{% else %}product--no-media{% endif %}">
+    <div class="featured-product isolate product grid grid--1-col gradient color-{{ section.settings.color_scheme }} {% if product.media.size > 0 %}grid--2-col-tablet{% else %}product--no-media{% endif %}">
```

Comment:
@kmeleta I think since we are allowing all of them to overlap I think maybe it makes sense to do so for consistency 🤔 

What do you think? 

cc @ludoboludo @tyleralsbury

Reply:
I kinda don't mind it in this situation with the secondary background setup 🤔  but i guess it'd be logical that the shadow goes over it. And again, what are the chances people are going to do no space like this and and vertical offset to the max 🤔  🤷